### DISCUSSION TITLE
Notary integration

### DISCUSSION THREAD

**Discussion by jvaidyan** [sse-secure-systems/connaisseur#326] [Q&A] [answered]
The admission controller has been deployed with default values. 
1) How do we integrate with an internal Notary server?
2) My understanding is we need to configure this as validator. Does this require a reinstall of Admission controller with updated values file?

**Comment by xopham** [comment, marked as answer]
Hey :wave:
1. Information on how to configure notary servers is contained in [the notary section](https://sse-secure-systems.github.io/connaisseur/v2.1.2/validators/notaryv1/)
2. Yes. Currently, you will need to update the `helm/values.yaml`, uninstall Connaisseur and install again. This will be much improved in the next release though.

**Reply by xopham** [reply]
Let me know if that helps, answered your question or you see need to improve the docs :slightly_smiling_face: 
Feel free to mark my answer as correct if that was the case :heavy_check_mark:
